From e925f0d35d8fb92963924f617c5deac208f751d7 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Sun, 23 Aug 2026 19:21:28 +0800
Subject: [PATCH 1/3] AOSCOS: create-config.mk: do not enable
 -fstack-protector-strong in build system

Architectures like DEC Alpha (alpha) does not have SSP support at all but
still advertises this option, only to throw an error.

Drop this option to save hassle.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 create-config.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/create-config.mk b/create-config.mk
index 3c3ba069..0d191464 100644
--- a/create-config.mk
+++ b/create-config.mk
@@ -187,7 +187,6 @@ $(TOPDIR)/config.mk:	$(multipathdir)/autoconfig.h
 	@echo "D_URCU_VERSION := $(call URCU_VERSION)" >>$@
 	@echo "SYSTEMD := $(SYSTEMD)" >>$@
 	@echo "ANA_SUPPORT := $(ANA_SUPPORT)" >>$@
-	@echo "STACKPROT := $(call TEST_CC_OPTION,-fstack-protector-strong,-fstack-protector)" >>$@
 	@echo "ERROR_DISCARDED_QUALIFIERS := $(call TEST_CC_OPTION,-Werror=discarded-qualifiers,)" >>$@
 	@echo "WNOCLOBBERED := $(call TEST_CC_OPTION,-Wno-clobbered -Wno-error=clobbered,)" >>$@
 	@echo "WFORMATOVERFLOW := $(call TEST_CC_OPTION,-Wformat-overflow=2,)" >>$@
-- 
2.55.0

